lesson5.rb

```diff
@@ -0,0 +1,19 @@
+#use loops to print the size of all strings
+names.map(&:size)
+
+#make all the selected string downcase
+names = names.map(&:downcase)
+
+#using the last array,create a hash with key:name and value:name size
+hash = {}
+hash = names.each_with_object({}) do |element, object| object[element] = element.size
+end
```

Comment:
This does the trick, but please separate the block with a newline or write it in one line:
`hash = names.each_with_object({}) { |element, object| object[element] = element.size }`
or
`hash = names.each_with_object({}) do |element, object|`
`object[element] = element.size`
`end`